deploy/esignet/install.sh

```
@@ -61,64 +109,290 @@ function installing_esignet() {
-    ESIGNET_HELM_ARGS="--set persistence.enabled=true  \
-            --set volumePermissions.enabled=true \
-            --set persistence.size=$volume_size \
-            --set persistence.mountDir=\"$volume_mount_path\" \
-            --set persistence.pvc_claim_name=\"$PVC_CLAIM_NAME\"  \
-            --set extraEnvVarsCM={'esignet-global','config-server-share','artifactory-share'} \
-            --set extraEnvVarsAdditional.MOSIP_KERNEL_KEYMANAGER_HSM_KEYSTORE-TYPE=PKCS12 \
-            "
+    ESIGNET_HELM_ARGS=(
+      --set persistence.enabled=true
+      --set volumePermissions.enabled=true
+      --set persistence.size=$volume_size
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)